src/platform/Zephyr/BLEManagerImpl.cpp

```diff
@@ -233,35 +238,57 @@ struct BLEManagerImpl::ServiceData
     ChipBLEDeviceIdentificationInfo deviceIdInfo;
 } __attribute__((packed));
 
-CHIP_ERROR BLEManagerImpl::StartAdvertising(void)
+inline CHIP_ERROR BLEManagerImpl::PrepareAdvertisingRequest()
```

Comment:
Why `inline` here?

Reply:
This is a way to tell the compiler to use internal linkage for a method. I hope it can help compiler make the decision to inline this function (didn't check it though).